plugins/single_plugins/expo.cpp

```diff
@@ -146,6 +209,11 @@ class wayfire_expo : public wf::plugin_interface_t
         output->render->set_renderer(renderer);
         output->render->set_redraw_always();
 
+        for (size_t i = 0; i < keyboard_select_cbs.size(); i++)
+        {
+            output->add_activator(keyboard_select_options[i], &keyboard_select_cbs[i]);
```

Comment:
This is a nice idea but to be frank, I don't care for this solution. It means you have to add a binding per workspace and that can get ridiculous quick. Instead, I think it might be better if there were prev/next workspace bindings or some binding with arrows to move around the workspace selection. Also click-to-select-workspace could be implemented, maybe double-click to select and end expo and/or you could have scroll wheel bound to prev/next.

Reply:
This is a more general issue which I am going to resolve in a next version of our config library. This approach is fine until then.